registries/agent_network_designer.hocon

```diff
@@ -150,15 +144,7 @@ Make agents granular so that you have at least a depth of three agents from the
                     "required": ["agent_name", "instructions"]
                 }
             },
-            "instructions": """
-{instructions_prefix}
-You add an agent to an agent network.
```

Comment:
That part should go in the `description` of the function. Currently it says:
```
I can generate the agent network hocon definition."
```
Instead of:
```
Adds an agent to an agent network.
```
The example is probably not needed.
The parameters specs should be enough.